src/test/test_random_walk.py

```diff
@@ -49,51 +41,55 @@ def test_rw_can_be_sampled():
     assert ans_rand[0].value[0] != init_rv_fixed.value
 
 
-def test_rw_samples_correctly_distributed():
+@pytest.mark.parametrize(
+    ["step_mean", "step_sd"],
+    [
+        [0, 1],
+        [2.253, 0.025],
+        [-3.2521, 1],
+        [1052, 3],
+        [1e-6, 0.02],
+    ],
+)
```

Comment:
This is fine. Just wondering what kinds of failure modes might be identified by trying these different combinations?

Reply:
- Failures of integer coercion to float (which should work)
- Failures to work with negative step means (which should work)
-  Very large and very small values don't cause problems (they shouldn't)

Reply:
I don't really see how we could introduce any of those issues, but I suppose its fine to test. These are just straightforwardly passed to `dist.Normal`, so none of our code attempts to check / process these arguments.